test/test-unpatch.js

```diff
@@ -16,22 +16,37 @@
 
 'use strict';
 
+require('./plugins/common.js');
 var assert = require('assert');
+var nock = require('nock');
+var nocks = require('./nocks.js');
 var trace = require('..');
 
-var common = require('./plugins/common.js');
-
 describe('index.js', function() {
   var agent;
+  var scope;
   var checkUnpatches = [];
+  before(function() {
+    delete process.env.GCLOUD_PROJECT;
+    nock.disableNetConnect();
```

Comment:
Can this be done at the top-level? That has a better chance of catching any early automatic requests as well.